examples/k8s/scale_server/README.md

```diff
@@ -0,0 +1,63 @@
+# Scaling SPIRE server
+
+## Configuration
+
+There are a few important consideration when scaling out the number of SPIRE
+servers for availability and resiliency.
+
+### Database Persistence
+
+To scale the number of SPIRE servers, an external datasource must be used to
+maintain shared state between servers. Currently, SPIRE supports using the
+Postgres database.
+
+### Topology
+
+The SPIRE server can be deployed either stateless or stateful, however in
+either case it's imperative to configure the server appropriately to handle
+failures and scalability.
+
++ **stateless** - To run the SPIRE server stateless, the `UpstreamCA` plugin
+  needs to be used, and `upstream_bundle = true` needs to be set. An example of
+  this configuration is the [postgres example](../postgres).
++ **stateful** - To run the SPIRE server stateful, the directory specified in
+  `data_dir` must be persistent, where a StatefulSet and PersistentVolumeClaim
+  are used. The [simple sat example](../simple_sat) is a partial example of this
+  configuration, missing Postgres for a datatore.
+
+#### Stateless Configuration
+
+In a stateless deployment, an external UpstreamCA is used and present in the
+bundle for the certificate verification chain. With this mode of deployment,
+any new server is trusted by SPIRE agents by verifying the certificate chain
+back to the UpstreamCA.
+
+Upstream CA Certificates should have a short TTL and be rotated to maintain
```

Comment:
Did you mean Upstream CA Certificates here? Or the intermediate certificates that are maintained by SPIRE server and fulfilled via UpstreamCA plugin?

Reply:
I'm not sure what I mean :-) I can use help with the correct terminology and details here.

Reply:
How about `CA certificates obtained via the UpstreamCA plugin should have a short TTL and be...`

Reply:
Thanks, I'll make that edit - and please nit-pick any of the other wording throughout, I appreciate it!